content/support/account-quotas/index.md

```diff
@@ -50,15 +50,7 @@ If the account usage is showing that quota is exceeded -[over 250 GB](/main/):
 * **Histories in a *deleted* state are still part of your quota usage.**
 * **Histories must be *purged* (permenently deleted) to not count toward quota.**
 
-### Find all Datasets or just review those in the active history and *purge* (aka permenently delete) those not needed
-
-* All account Datasets can be reviewed under **User > Datasets**.
-* Within a History, deleted/permenently deleted Datasets can be reviewed by toggling the *deleted* link at the top of the History panel, found immediately under the History name.
-* Both *active* (shown by default) and *hidden* (the other toggle link, next to the *deleted* link) datasets can be reviewed the same way.
-* Click on the far right "X" to delete a dataset.
-* To permenently delete: use the link from within the dataset, or use the *Operations on Multiple Datasets* functions, or use the *Purge Deleted Datasets* option in the History menu.
-* **Datasets in a *deleted* state are still part of your quota usage.**
-* **Datasets must be *purged* (permenently deleted) to not count toward quota.**
+### [Find all Datasets or just review those in the active history and *purge* (aka permenently delete) those not needed](https://training.galaxyproject.org/training-material/faqs/galaxy/datasets_purgin_datasets.html)
```

Comment:
```suggestion
### [Find all Datasets or just review those in the active history and *purge* (aka permenently delete) those not needed](https://training.galaxyproject.org/training-material/faqs/galaxy/datasets_purging_datasets.html)
```

changed the file name [here](https://github.com/galaxyproject/training-material/pull/3418)